halyard-web/lib/systemd/system/halyard.service

```diff
@@ -4,9 +4,9 @@ Description=A tool for managing the lifecycle of a Spinnaker installation
 [Service]
 Restart=always
 RestartSec=10
-Type=forking
-ExecStart=
-ExecStart=sudo -u $(cat /var/spinnaker/config/halyard-user) -g $(cat /var/spinnaker/config/halyard-user) /opt/halyard/bin/halyard 2>&1 > /var/log/spinnaker/halyard/halyard.log &
+Type=simple
+ExecStart=/opt/halyard/bin/halyard
```

Comment:
Looks good, where does Halyard log to by default? stdout?

Do we still need this: `/opt/halyard/bin/halyard 2>&1 > /var/log/spinnaker/halyard/halyard.log` ?

Reply:
Or does `journald` pick it up?

Reply:
The systemd syntax does dot allow redirection of output to a log file, redirecting the output to the log file was one of the issues that was causing the service not to start.

https://manpages.ubuntu.com/manpages/bionic/en/man5/systemd.service.5.html

```
 This syntax is inspired by shell syntax, but only the meta-characters and expansions
       described in the following paragraphs are understood, and the expansion of variables is
       different. Specifically, redirection using "<", "<<", ">", and ">>", pipes using "|",
       running programs in the background using "&", and other elements of shell syntax are not
       supported.
       ```
       
 But as you mentioned, journald picks it up and the logs can be viewed using `journalctl`.